src/client/pydaos/raw/daos_cref.py

```diff
@@ -99,6 +99,31 @@ class PoolInfo(ctypes.Structure):
                 ("pi_space", PoolSpace),
                 ("pi_rebuild_st", RebuildStatus)]
 
+
+class DaosPropertyEntry(ctypes.Structure):
+    _fields_ = [("dpe_type", ctypes.c_uint32),
+                ("dpe_reserv", ctypes.c_uint32),
+                ("dpe_val", ctypes.c_uint64)]
+
+
+class DaosProperty(ctypes.Structure):
```

Comment:
(pylint-missing-docstring) Missing class docstring
(pylint-too-few-public-methods) Too few public methods (0/2)

Reply:
Entire file has this problem.